Modules/Filtering/Path/test/itkContourExtractor2DImageFilterTest.cxx

```diff
@@ -468,6 +572,13 @@ itkContourExtractor2DImageFilterTest(int argc, char * argv[])
     return EXIT_FAILURE;
   }
 
+  // verify that the default ReverseContourOrientation is false
+  if (extractor->GetReverseContourOrientation() != false)
```

Comment:
There are some macros in itkTestingMacros.h that may be helpful here.

Reply:
They will be used.